main.go

```diff
@@ -39,8 +39,7 @@ func main() {
 	)
 
 	if err != nil {
-		fmt.Fprintln(os.Stderr, err)
-		os.Exit(1)
+		logger.Error("An error occurred:", err)
```

Comment:
If err != nil here, then logger is going to be nil, and this will panic.  I think what was there before - logging to stderr and exiting 1 - is still probably the right move, but I'd add some additional context there.  So like: `fmt.Fprintf(os.Stderr, "Unable to initialize the logger: %s", err)`

Reply:
ok yup i slipped this in being like 'hm should probably use the logger huh'
cant use the logger if it doesnt exist hah